config/setup

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
just had another idea, could this script check if the google-java-format-1.24.0-all-deps.jar exists and of not, download it with curl - curl -LJO https://github.com/google/google-java-format/releases/download/v1.24.0/google-java-format-1.24.0-all-deps.jar

Reply:
then we can remove the jar from our git repo

Reply:
and use the version var in the curl

Reply:
updated
download jar file, copy to hooks folder, remove it from config